### PR TITLE
Custom objects: Create object positive and negative cases

### PR DESCRIPTION
**Pull request by annamharika**
Create custom object cases

### CODE REVIEW

**Comment by TejasreeMarthy** [comment]
Resolve conflicts

**Comment by annamharika** [comment]
Conflicts resolved

File: Tests/Android_Studio/CustomObject/Create.Object.Positive.test.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+			.elementById('com.example.axway.mbaas:id/customobjects_create_new_property_key_field')
+			.sendKeys(customObject.propertyKey) // binding static information to input fields to create a property key
+			.elementById('com.example.axway.mbaas:id/customobjects_create_new_property_key_field')
+			.text().should.become(customObject.propertyKey);
+	});
+	it('Add new property value and Create a Custom Object', () => {
+		return driver
+			.elementById('com.example.axway.mbaas:id/customobjects_create_add_new_property_button')
+			.click()
+			.elementByAndroidUIAutomator('new UiSelector().text("New Property Key")')
```
Comment (SamJHaig) [inline Tests/Android_Studio/CustomObject/Create.Object.Positive.test.js]: Here you are filling out the new property key field again, instead you want to be filling in the field for the property that was just created, in this case "UserName". At the moment the UserName value is empty at the time of creating the object

File: Tests/Android_Studio/CustomObject/Create.Object.Positive.test.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+			.sendKeys(customObject.propertyValue) // binding static information to input fields to create a property value
+			.elementByAndroidUIAutomator(`new UiSelector().text("${customObject.propertyValue}")`)
+			.text().should.become(customObject.propertyValue);
```
Comment (SamJHaig) [inline Tests/Android_Studio/CustomObject/Create.Object.Positive.test.js]: These two lines are doing the same thing really, as you wouldn't be able to find the element without the text being what we're expecting. Instead I would recommend doing:
```
.elementByAndroidUIAutomator(`new UiSelector().text("${customObject.propertyValue}")`)
.isDisplayed().should.become(true);
```
or:
```
.elementByXPath('//android.widget.EditText[@instance="2"]')
.text().should.become(customObject.propertyValue);
```

Comment (annamharika) [reply Tests/Android_Studio/CustomObject/Create.Object.Positive.test.js]: Modified above change in latest commit

File: Tests/Android_Studio/CustomObject/Create.Object.Negative.test.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+			.text().should.become(customObject.className);
+	});
+	it('Enter a new property key', () => {
+		return driver
+			.elementById('com.example.axway.mbaas:id/customobjects_create_new_property_key_field')
+			.sendKeys(customObject.propertyKey) // binding static information to input fields to create a property key
+			.elementById('com.example.axway.mbaas:id/customobjects_create_new_property_key_field')
+			.text().should.become(customObject.propertyKey);
+	});
+	it('Add new property and Create a Custom Object', () => {
```
Comment (SamJHaig) [inline Tests/Android_Studio/CustomObject/Create.Object.Negative.test.js]: You should also be doing the 'Add new property value and Create a Custom Object' step before this one

File: Tests/Android_Studio/CustomObject/Create.Object.Negative.test.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+		return driver
+			.elementById('com.example.axway.mbaas:id/customobjects_create_add_new_property_button')
+			.click()
+			.elementById('com.example.axway.mbaas:id/customobjects_create_create_button')
+			.click() // It creates a custom object
+			.waitForElementById('android:id/message', webdriver.asserters.isDisplayed, 10000)
+			.getAttribute('text')
+			.then(text => {
+				text.should.include('"status":"fail"');
+				text.should.include('"code":400');
```
Comment (SamJHaig) [inline Tests/Android_Studio/CustomObject/Create.Object.Negative.test.js]: Please also validate the "method_name" and "message"

Comment (annamharika) [reply Tests/Android_Studio/CustomObject/Create.Object.Negative.test.js]: Added "message" validation in latest commit, "method_name" is not getting from the server

File: Tests/Android_Studio/CustomObject/Create.Object.Negative.test.js (hunk 3 above)
Comment (annamharika) [reply Tests/Android_Studio/CustomObject/Create.Object.Negative.test.js]: Modified above change mentioned in latest commit

**Pull request merged**